conf/ocsci/ocp-4.3-ga-upgrade.yaml

```diff
@@ -0,0 +1,5 @@
+---
+# Use this config for upgrading of OCP 4.3 GA cluster
+UPGRADE:
+  ocp_upgrade_version: "4.3.3-x86_64"
```

Comment:
As suggested in other comment I would go with architecture in different value and I think with using my function we can delete this value and use dynamically latest based on channel which you can define here.